### PR TITLE
Small fix in kubelet.md

### PR DESCRIPTION
**Pull request by hs0210**
Signed-off-by: Hu Shuai <hus.fnst@fujitsu.com>

### CODE REVIEW

File: content/en/docs/reference/command-line-tools-reference/kubelet.md [hunk 1]
```diff
@@ -1091,7 +1091,7 @@ WindowsHostProcessContainers=true|false (BETA - default=true)<br/>
 </tr>
 <tr>
 <td></td><td style="line-height: 130%; word-wrap: break-word;">Comma-separated list of cipher suites for the server. If omitted, the default Go cipher suites will be used.<br/>
-Preferred values:
+Preferred values:<br/>
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/kubelet.md:1094]: Why this change?

File: content/en/docs/reference/command-line-tools-reference/kubelet.md [hunk 2]
```diff
@@ -219,7 +219,7 @@ kubelet [flags]
 <td colspan="2">--container-runtime-endpoint string</td>
 </tr>
 <tr>
-<td></td><td style="line-height: 130%; word-wrap: break-word;">The endpoint of remote runtime service. Unix Domain SOckets are supported on Linux, while npipe and tcp endpoints are supported on windows. Examples: <code>unix:///var/run/dockershim.sock</code>, <code>npipe:////./pipe/dockershim</code>.</td>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">The endpoint of remote runtime service. Unix Domain Sockets are supported on Linux, while npipe and tcp endpoints are supported on windows. Examples: <code>unix:///var/run/dockershim.sock</code>, <code>npipe:////./pipe/dockershim</code>.</td>
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/kubelet.md:222]: I checked this against https://github.com/kubernetes/kubernetes/blob/release-1.24/cmd/kubelet/app/options/options.go#L302

```suggestion
<td></td><td style="line-height: 130%; word-wrap: break-word;">The endpoint of remote runtime service. Unix Domain Sockets are supported on Linux, while npipe and tcp endpoints are supported on windows. Examples: <code>unix:///path/to/runtime.sock</code>, <code>npipe:////./pipe/runtime</code>.</td>
```

**Review by sftim — COMMENTED** [review]
We need to make sure that any update is taking the correct text from upstream code.

File: content/en/docs/reference/command-line-tools-reference/kubelet.md (hunk 1 above)
Comment (hs0210) [reply content/en/docs/reference/command-line-tools-reference/kubelet.md:1094]: Reference [Insecure values](https://github.com/kubernetes/website/commit/01e0a9ebd16fb7650df82ae9253f19f25e0caa95#diff-a7a37ad9496edf0e4b1ef0cf4270571e43d83d29a9b8fd9c14b6a155a6746e38R1096)

Comment (hs0210) [reply content/en/docs/reference/command-line-tools-reference/kubelet.md]: I checked against the [upstream code](https://github.com/kubernetes/kubernetes/blob/release-1.24/cmd/kubelet/app/options/options.go#L412-L413), no newline after `Preferred values:` and `Insecure values:`. I have deleted the `<br/>` after `Preferred values:` and `Insecure values:`. PTAL.

File: content/en/docs/reference/command-line-tools-reference/kubelet.md [hunk 3]
```diff
@@ -1091,9 +1091,9 @@ WindowsHostProcessContainers=true|false (BETA - default=true)<br/>
 </tr>
 <tr>
 <td></td><td style="line-height: 130%; word-wrap: break-word;">Comma-separated list of cipher suites for the server. If omitted, the default Go cipher suites will be used.<br/>
-Preferred values:
+Preferred values:
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/kubelet.md:1094]: Please omit this whitespace change.

File: content/en/docs/reference/command-line-tools-reference/kubelet.md [hunk 4]
```diff
@@ -1091,9 +1091,9 @@ WindowsHostProcessContainers=true|false (BETA - default=true)<br/>
 </tr>
 <tr>
 <td></td><td style="line-height: 130%; word-wrap: break-word;">Comma-separated list of cipher suites for the server. If omitted, the default Go cipher suites will be used.<br/>
-Preferred values:
+Preferred values: 
 `TLS_AES_128_GCM_SHA256`, `TLS_AES_256_GCM_SHA384`, `TLS_CHACHA20_POLY1305_SHA256`, `TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA`, `TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256`, `TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA`, `TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384`, `TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305`, `TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256`, `TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA`, `TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256`, `TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA`, `TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384`, `TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305`, `TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256`, `TLS_RSA_WITH_AES_128_CBC_SHA`, `TLS_RSA_WITH_AES_128_GCM_SHA256`, `TLS_RSA_WITH_AES_256_CBC_SHA`, `TLS_RSA_WITH_AES_256_GCM_SHA384`<br/>
-Insecure values:<br/>
+Insecure values:
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/kubelet.md:1096]: Please omit the trailing space here.

**Review by sftim — COMMENTED** [review]
/lgtm
(with minor, non-blocking, concerns)

File: content/en/docs/reference/command-line-tools-reference/kubelet.md (hunk 3 above)
Comment (hs0210) [reply content/en/docs/reference/command-line-tools-reference/kubelet.md]: Done. PTAL.

**Review by Sea-n — COMMENTED** [review]
/lgtm
